code/modules/mob/inventory.dm

```diff
@@ -186,9 +186,9 @@
 /mob/proc/put_in_hand_check(obj/item/I)
 	return FALSE //nonliving mobs don't have hands
 
-/mob/living/put_in_hand_check(obj/item/I)
-	if(istype(I) && ((mobility_flags & MOBILITY_PICKUP) || (I.item_flags & ABSTRACT)) \
```

Comment:
This feels like poking at a wasps nest, but this seems like a mistake that allows for the pick-up of abstract objects